resources/rgbscripts/checkers.js

```diff
@@ -0,0 +1,360 @@
+/*
+  Q Light Controller Plus
+  checkers.js
+
+  Copyright (c) Branson Matheson
+  Based on the work of Massimo Callegari and Tim Cullingworth
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0.txt
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// Development tool access
+var testAlgo;
+
+(
+  function()
+  {
+    var colorPalette = new Object;
+    colorPalette.collection = new Array(
+      ["White"        , 0xFFFFFF],  //  0
+      ["Cream"        , 0xFFFF7F],  //  1
+      ["Pink"         , 0xFF7F7F],  //  2
```

Comment:
Codacy has a fix for the issue: [Replace `······["Pink"·········,·0xFF7F7F],·` with `····["Pink",·0xff7f7f],`](https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=9305479)
```suggestion
    ["Pink", 0xff7f7f], //  2
```